### ISSUE TITLE
Action dropdown title

### ISSUE THREAD

**Issue by marcoscv-work** [liferay/clay#1161]
When action dropdown is associated to an item its title should be "**Name of item** Actions".

An example in a card:

![image](https://user-images.githubusercontent.com/7963804/44982697-0edaf600-af77-11e8-9ad0-3568d2471f64.png)

![image](https://user-images.githubusercontent.com/7963804/44982723-21edc600-af77-11e8-8515-9b846e42f678.png)

In a list:

![image](https://user-images.githubusercontent.com/7963804/44982821-6f6a3300-af77-11e8-94cd-b329fa0d46b0.png)

![image](https://user-images.githubusercontent.com/7963804/44982867-8f99f200-af77-11e8-8dc2-66e005a02712.png)

**Comment by travisrcory** [comment]
I'm looking into this!

**Comment by travisrcory** [comment]
https://github.com/liferay/clay/pull/1516

**Comment by matuzalemsteles** [comment]
Just adding a progress on this: I've had to reverse Travis change in ClayList, the Actions have no view of the information of the columns of row, this is difficult to identify since the value of the title can be in any column.

cc @carloslancha

**Comment by carloslancha** [comment]
This is solved then or any more work is needed? @matuzalemsteles

**Comment by matuzalemsteles** [comment]
hey @carloslancha still lacking work to be done in ClayList.

**Comment by jbalsas** [comment]
Doing some spring-cleaning... closing all issues labeled as 2.x that are not type:bug since 2.x is in maintenance and we won't be proactively be taking these ones.

@matuzalemsteles, @carloslancha, please reopen if you think this is still relevant in the context of our `3.x` components.

Thanks!
